tasks.js

```diff
@@ -12,8 +12,16 @@ Add functionality to getTasks method so that it
 
 const getTasks = async (id) => {
   /* only change the implementation of this function */
+  fetch(`https://mod2-api.herokuapp.com/owner/${id}/task`)
```

Comment:
Nice work!

The only change i would make is to remove the async keyword from in front of the function as it is not needed when you don't use await inside of the function

Reply:
Understood will make the change, thank you.

Reply:
So I referred to class, to see what would be required if the async was there with the await and also removed the .then. Just so I get the hang of it's syntax.

const getTasks = async (id) => {
  const response = await fetch(`https://mod2-api.herokuapp.com/owner/${id}/task`)
  const data = response.json()
  tasks = data;
  renderTasks()
}